Jenkinsfile

```diff
@@ -308,6 +309,56 @@ def buildDiffCoverage() {
     }
 }
 
+def buildPerformance() {
+  return {
+    // Select docker-cph-X.  We want docker, metal (brix) and only one executor
+    // (exclusive), if the machine changes also change REALM_BENCH_MACHID below
+    node('docker && brix && exclusive') {
+      getSourceArchive()
+
+      def gitTag = readGitTag()
+      def gitSha = readGitSha()
+
+      if (gitTag == "") {
+        setBuildName(gitSha)
```

Comment:
I don't think we should set the build name in here. It's already done at the beginning of the job. It looks like lines 322 to 326 could be removed.